llvm/labs/lab2/kosarev_egor/kosarevInlinePass.cpp

```diff
@@ -0,0 +1,172 @@
+#include "llvm/ADT/SmallPtrSet.h"
```

Comment:
Please add an option `-no-loop-inline` which turns off inlining of functions that contain loops.

Reply:
done